RCTVideo.m

```diff
@@ -18,8 +17,8 @@ @implementation RCTVideo
 {
   AVPlayer *_player;
   AVPlayerItem *_playerItem;
+  AVPlayerViewController *_playerLayer;
   BOOL _playerItemObserverSet;
-  AVPlayerLayer *_playerLayer;
```

Comment:
I am planning on cleaning this up as follows: 
- I'll introduce a new property called `controls` that mirrors the `video` tag's property (http://www.w3schools.com/tags/att_video_controls.asp)
- If `controls` is set I'll use `AVPlayerViewController` otherwise I'll use `AVPlayerLayer`
That would make the most sense to me.